task2/src/main/java/ElectricalAppliances/CleaningDevice/CleaningDevice.java

```diff
@@ -0,0 +1,20 @@
+package ElectricalAppliances.CleaningDevice;
+
+import ElectricalAppliances.ElectricalAppliances;
+
+
+public abstract class CleaningDevice extends ElectricalAppliances {
```

Comment:
What the reason that this class should be abstract?
If you want to prevent the creation of an object of this class you should constructor Access modifier to 'private'